home-application/pom.xml

```diff
@@ -111,7 +88,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-dependency-plugin</artifactId>
-                <version>2.8</version>
+                <version>3.1.2</version>
```

Comment:
Versions should be in a separate section

Reply:
Done.